### DISCUSSION TITLE
Widget Improvements - Time Checked and Refresh Icon

### DISCUSSION THREAD

**Discussion by hl2guide** [breezy-weather/breezy-weather#1337] [Ideas]
### Describe your idea

Please improve widgets showing:

- time last checked
- refresh icon to refresh stale data directly on the widget

Please refer to Samsung Weather as an example for inspiration.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
> time last checked

Choose "Refresh time" as your option for "Subtitle data"

> refresh icon to refresh stale data directly on the widget

> - [x] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).

> Features that will not be implemented
> - Changes to the [background updates process](https://github.com/breezy-weather/breezy-weather/blob/main/docs/UPDATES.md), including but not limited: options for refreshing less than every 30 minutes, every time you open the app, every time you tap on widget, every time you unlock your phone
